.github/workflows/publish.yml

```diff
@@ -0,0 +1,53 @@
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '14.x'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Install dependencies and build 🔧
+        run: npm ci && npm run build
+      - name: Run lint 
+        run: npm run lint
+      - name: Run tests 
+        run: npm run test
```

Comment:
@brianignacio5 should we run build & lint & test steps also on pull requests? (And package etc only for tags)